sql_upgrade.php

```diff
@@ -307,6 +311,7 @@ function pausePoll(othis) {
                 flush_echo("<script>serverStatus(" . js_escape($version) . ", 1);</script>");
                 upgradeFromSqlFile($filename);
                 // end polling
+                sleep(2); // fixes odd bug, where if the sql upgrade goes to fast, then the polling does not stop
```

Comment:
@sjpadgett ,
Odd bug here where the polling never stops if the sql upgrade is too quick. Adding this (ie. waiting for a couple seconds) seems to fix the issue. Gets even crazier if polling stays on and try to run the patch upgrade below (which is why incorporated a poll kill parameter there just in case the above sleep doesn't fix the poll stopping bug in other environments).

Reply:
Yeah, if ajax process misses the kill when php moves on then, I can see this happening. There will always be an unknown delay before the long poll returns from masking. It really needs a timer to time out if progress isn't updated.

I may have overdid this somewhat but, I don't think I could have been stopped at the time!:)

Reply:
yep, I suffer from the same disease :)
Very nice feature, though.
For the future, we could do what happens at the gas pump and play video ads in the upgrade screen until it's done :)

Reply:
lol, I like that idea! However Brady, you really shouldn't put thoughts like that in my head. Never know when a PR will pop up with that craziness.:)
Or we could display our minions gif on final!!!